library/platform_util.c

```diff
@@ -65,3 +74,54 @@ void mbedtls_platform_zeroize( void *buf, size_t len )
     memset_func( buf, 0, len );
 }
 #endif /* MBEDTLS_PLATFORM_ZEROIZE_ALT */
+
+#if defined(MBEDTLS_HAVE_TIME_DATE) && !defined(MBEDTLS_PLATFORM_GMTIME_R_ALT)
+#include <time.h>
+#if !defined(_WIN32) && (defined(unix) || \
+    defined(__unix) || defined(__unix__) || (defined(__APPLE__) && \
+    defined(__MACH__)))
+#include <unistd.h>
+#if !defined(_POSIX_VERSION) || _POSIX_C_SOURCE > _POSIX_THREAD_SAFE_FUNCTIONS
```

Comment:
This condition is actually not correct: recent POSIX versions have moved `gmtime_r` from the Thread-Safe Functions option to the base. I think the correct condition is:
```
#if _POSIX_VERSION >= 200809L_|| POSIX_THREAD_SAFE_FUNCTIONS >= 200112L
#define MBEDTLS_HAVE_GMTIME_R
#endif
```

Reply:
Thanks, I'll correct it.

Reply:
Fixed in 6f70581